integrate/data-streams/overview/index.md

```diff
@@ -25,18 +25,35 @@ Events are received using HTTP. Receivers are associated with a maximum of 20 `e
 
 The available column types are:
 
-|Type|Description|
-|:-|:-|
-| `id`| event ID |
-| `datetime` | time of the event |
-| `ip` | IP of the event sender |
-| `body` | the unaltered event body |
-| `headers` | the unaltered request headers |
+|Type| Description                                |
+|:-|:-------------------------------------------|
+| `id`| event ID                                   |
+| `datetime` | time of the event                          |
+| `ip` | IP of the event sender                     |
+| `body` | the unaltered event body                   |
+| `headers` | the unaltered request headers              |
+| `path` | a field from json object                   |
 | `template` | a custom mapping using a template language |
 
+### Path
+
+The `path` column type can be used to fetch a single field from a `json` object. Optionally you can use `rawString` option to remove the quotes around a json string or a `defaultValue` option to define a value when the field doesn't exist.
+
+```json
+{
+  "type": "json",
+  "name": "id",
+  "path": "issue.id",
+  "defaultValue": "undefined", 
+  "rawString": true
+}
+```
+
 ### Template (Jsonnet)
 
-The `template` column type currently only supports the `jsonnet` templating language.The following `jsonnet` globals are available:
+Note: It is recommended to use the much faster `path` type instead of `jsonnet` function `Body(string)` when possible. 
```

Comment:
We use `path` only when accessing fields or? Should this be mentioned in `template`?